### PR TITLE
Removing AABB.

### PR DESCRIPTION
**Pull request by unpacklo**
https://unity3d.atlassian.net/browse/DOTS-1697

### CODE REVIEW

File: src/Unity.Mathematics/Geometry/MinMaxAABB.cs [hunk 1]
```diff
@@ -198,18 +198,6 @@ public void Encapsulate(float3 point)
             Max = math.max(Max, point);
         }
 
-        [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        public static implicit operator MinMaxAABB(AABB aabb)
```
Comment (bkaradzic-unity3d) [inline src/Unity.Mathematics/Geometry/MinMaxAABB.cs:202]: If this is released to public you need to add:

`[Obsolete("<what happend> (RemovedAfter <today + 3 months in yyyy-mm-dd format>).")]`

File: src/Unity.Mathematics/Geometry/MinMaxAABB.cs [hunk 2]
```diff
@@ -198,18 +198,6 @@ public void Encapsulate(float3 point)
...
-        {
-            return new MinMaxAABB {Min = aabb.Center - aabb.HalfExtents, Max = aabb.Center + aabb.HalfExtents};
-        }
-
-        [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        public static implicit operator AABB(MinMaxAABB aabb)
```
Comment (bkaradzic-unity3d) [inline src/Unity.Mathematics/Geometry/MinMaxAABB.cs:208]: If this is released to public you need to add:

`[Obsolete("<what happend> (RemovedAfter <today + 3 months in yyyy-mm-dd format>).")]`

File: src/Unity.Mathematics/Geometry/MinMaxAABB.cs (hunk 1 above)
Comment (unpacklo) [reply src/Unity.Mathematics/Geometry/MinMaxAABB.cs:202]: All of this stuff is internal so I don't think I need to mark this as obsolete yet, but there are versions of this code in Entities which I plan to delete, so I probably need to mark as obsolete over there?

**Comment by AdamFrisby** [comment]
Hi guys - this broke some code, uh, where did it go? (That JIRA instance is private)

**Comment by unpacklo** [comment]
> Hi guys - this broke some code, uh, where did it go? (That JIRA instance is private)
> 
> Edit: uh, maybe not this particular, but I can't seem to find a UnityEngine.Mathematics.AABB in my project for some reason.

This change should not have broken anything in your code since we haven't done a release in quite some time. There is no such thing as `UnityEngine.Mathematics.AABB` but there does currently exist a `Unity.Mathematics.AABB` in the Entities package as an extension. If you have the Entities package then you should be able to use that AABB.

**Pull request merged**